maintainers/scripts/validate-commit-titles/validate.rs

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env nix-shell
+//! ```cargo
+//! [dependencies]
+//! regex = "1"
+//! ```
+
+use std::{fs::read_to_string, process::{Command, Stdio, ExitCode}};
+use regex::Regex;
+/*
+#!nix-shell -i rust-script -p rustc -p rust-script -p cargo -p gh -p jq --
+*/
+
+fn validate(titles: Vec<&str>) -> bool {
+    let re = Regex::new(r"^[^{]+:.+$").unwrap();
```

Comment:
Should `{a,b}: …` be invalid? I prefer it as it is more consistent with general glob syntax. Does ofborg not handle it or something?

Reply:
It's valid but this was written before ofborg supported brace expansion.

Reply:
Oh, I missed the `+` :)